### ISSUE TITLE
BUG: Gene rollup exception causes 500 error at query end point

### ISSUE THREAD

**Issue by prathapsridharan** [chanzuckerberg/single-cell-data-portal#4867]
# Steps to reproduce

1. Pick `organism: homo sapiens`
2. Pick 2 `Tissues: [spinal cord, liver]`
3. Pick 1 `Gene: MS4A1`

This happens when the genes and tissues picked on the frontend are such that one of the tissues has NO expression of any of the genes picked. As the simple example above suggests, if you pick 2 tissues and 1 gene, and that single gene is not expressed in one of the tissues, then that tissue has no gene expression at all within the query constraints.

### Request body
```
{"filter":{"dataset_ids":[],"development_stage_ontology_term_ids":[],"disease_ontology_term_ids":[],"gene_ontology_term_ids":["ENSG00000156738"],"organism_ontology_term_id":"NCBITaxon:9606","self_reported_ethnicity_ontology_term_ids":[],"sex_ontology_term_ids":[],"tissue_ontology_term_ids":["UBERON:0002240","UBERON:0002107"]},"is_rollup":true}
```

### Error stack trace from 

https://us-west-2.console.aws.amazon.com/cloudwatch/home?region=us-west-2#logsV2:log-groups/log-group/$252Fdp$252Fprod$252Fprodstack$252Fbackend/log-events/fargate$252Fweb$252F0c40d564d5504d808640c6941fba2d35$3Fstart$3D-1800000$26filterPattern$3D$2522ERROR$2522

Search for *"ERROR"* in the cloudwatch link above

```
{
    "levelname": "ERROR",
    "asctime": "2023-05-19T17:33:17.019Z",
    "name": "corpora-api-prod",
    "message": "InternalServerError",
    "lineno": 182,
    "pathname": "/single-cell-data-portal/backend/api_server/app.py",
    "request_id": "838b1559-bcfc-490c-8cf6-1c839f123fc5",
    "exc_info": "Traceback (most recent call last):\n  File \"/usr/local/lib/python3.10/dist-packages/flask/app.py\", line 1823, in full_dispatch_request\n    rv = self.dispatch_request()\n  File \"/usr/local/lib/python3.10/dist-packages/flask/app.py\", line 1799, in dispatch_request\n    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)\n  File \"/usr/local/lib/python3.10/dist-packages/connexion/decorators/decorator.py\", line 68, in wrapper\n    response = function(request)\n  File \"/usr/local/lib/python3.10/dist-packages/connexion/decorators/uri_parsing.py\", line 149, in wrapper\n    response = function(request)\n  File \"/usr/local/lib/python3.10/dist-packages/connexion/decorators/validation.py\", line 196, in wrapper\n    response = function(request)\n  File \"/usr/local/lib/python3.10/dist-packages/connexion/decorators/response.py\", line 112, in wrapper\n    response = function(request)\n  File \"/usr/local/lib/python3.10/dist-packages/connexion/decorators/parameter.py\", line 116, in wrapper\n    return function(**kwargs)\n  File \"/single-cell-data-portal/backend/wmg/api/v1.py\", line 65, in query\n    dot_plot_matrix_df, cell_counts_cell_type_agg = rollup(dot_plot_matrix_df, cell_counts_cell_type_agg)\n  File \"/single-cell-data-portal/backend/wmg/api/v1.py\", line 312, in rollup\n    dot_plot_matrix_df = add_missing_combinations_to_dot_plot_matrix(dot_plot_matrix_df, cell_counts_cell_type_agg)\n  File \"/single-cell-data-portal/backend/wmg/api/v1.py\", line 296, in add_missing_combinations_to_dot_plot_matrix\n    entry[\"n_cells_tissue\"] = n_cells_tissue_dict[entry[\"tissue_ontology_term_id\"]]\nKeyError: 'UBERON:0002240'"
}
```

**Comment by prathapsridharan** [comment]
![Screenshot 2023-05-19 at 11.33.01 AM.png](https://images.zenhubusercontent.com/640fa56ccc7351c0072bda12/d1f69b9e-ff50-43b9-a93f-64c3624131c1)

**Comment by atarashansky** [comment]
Addressed in #4869
